### PR TITLE
2.1 string review

### PR DESCRIPTION
**Pull request by Be-ing**
TODO: effects descriptions and parameters

### CODE REVIEW

**Comment by Be-ing** [comment]
Rebased on #1254 to ensure there are no merge conflicts with the effects tooltips

File: src/effects/native/echoeffect.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ EffectManifest EchoEffect::getManifest() {
     manifest.setShortName(QObject::tr("Echo"));
     manifest.setAuthor("The Mixxx Team");
     manifest.setVersion("1.0");
-    manifest.setDescription(QObject::tr("Stores the input signal in a temporary buffer"
+    manifest.setDescription(QObject::tr("Stores the input signal in a temporary buffer "
```
Comment (daschuer) [inline src/effects/native/echoeffect.cpp]: Why we need a space here?

Comment (Be-ing) [reply src/effects/native/echoeffect.cpp]: Because the string continues on the next line

Comment (daschuer) [reply src/effects/native/echoeffect.cpp]: This is a translators trap. Can we combine both strings? Or can we exclude the space  part from translations?

Comment (Be-ing) [reply src/effects/native/echoeffect.cpp]: Sure, we can combine the strings if you're okay with long lines of code.

Comment (Be-ing) [reply src/effects/native/echoeffect.cpp]: Are you sure the strings aren't put together on Transiflex? @esbrandt do you know?

**Comment by Be-ing** [comment]
Rebased on master for easier review

File: src/effects/native/echoeffect.cpp (hunk 1 above)
Comment (esbrandt) [reply src/effects/native/echoeffect.cpp]: It is fine, and used in different places. Strings inside `tr()` are treated as paragraph. So the translator will see the whole sentence here. Transifex even support translators with consistency checks and markers for invisible elements like space and carriage return.

Comment (Be-ing) [reply src/effects/native/echoeffect.cpp]: Thank you for clarifying. I reformatted the code so these awkward line breaks are not necessary anyway.

File: src/skin/tooltips.cpp [hunk 2]
```diff
@@ -783,7 +794,7 @@ void Tooltips::addStandardTooltips() {
 
     add("EffectSlot_enabled")
             << tr("Enable Effect")
-            << tr("This effect chain must also be assigned to a deck or other sound source to hear the effect.");
+            << tr("This effect unit must also be assigned to a deck or other sound source to hear the effect.");
```
Comment (esbrandt) [inline src/skin/tooltips.cpp]: For the ``EffectSlot_enabled`` control the text is about effect *unit*, same as with ``EffectUnit_``controls.
My understanding was ``racks -> unit -> slot``, has it changed?

File: src/effects/native/autopaneffect.cpp [hunk 3]
```diff
@@ -18,21 +18,22 @@ QString AutoPanEffect::getId() {
...
     EffectManifestParameter* period = manifest.addParameter();
     period->setId("period");
     period->setName(QObject::tr("Period"));
-    period->setDescription(QObject::tr("How fast the sound goes from a side to another\n"
-            "1/4 - 4 beats rounded to 1/2 beat if tempo is detected (decks and samplers)\n"
-            "1/4 - 4 seconds if no tempo is detected (mic & aux inputs, master mix)"));
+    period->setShortName(QObject::tr("Period"));
+    period->setDescription(QObject::tr(
+        "How fast the sound goes from one side to another\n"
+        "1/4 - 4 beats rounded to 1/2 beat if tempo is detected\n"
```
[the remark this answers is not in the document]
Comment (Be-ing) [reply src/effects/native/autopaneffect.cpp]: In reviewing #1254 we decided to use the master sync clock for mic, aux, and master mix. We'll implement that during the beta period.

File: src/skin/tooltips.cpp (hunk 2 above)
Comment (daschuer) [inline src/skin/tooltips.cpp]: The effect unit
(The effect unit this effect is part of)

Comment (Be-ing) [reply src/skin/tooltips.cpp]: rack: distinction in the code that makes no difference to users. IMO we should not mention it in user visible strings
chain: a group of effects processed in series
unit: the thing that a chain is loaded into

Comment (Be-ing) [reply src/skin/tooltips.cpp]: good point

Comment (daschuer) [reply src/skin/tooltips.cpp]: Ok, I will fix this after merge.

File: src/skin/tooltips.cpp [hunk 4]
```diff
@@ -267,6 +265,19 @@ void Tooltips::addStandardTooltips() {
             << tr("Hold-to-talk or short click for latching to")
             << tr("mix microphone input into the master output.");
 
+    add("talkover_duck_mode")
```
Comment (Be-ing) [inline src/skin/tooltips.cpp]: Note: These need to be added to all skins.

Comment (daschuer) [reply src/skin/tooltips.cpp]: There are also other persistent controls where, we need to make it either accessible or force it to a default after skin change.

**Pull request merged**